profile.php

```diff
@@ -302,34 +302,41 @@ function userProfile() {
 		<label style='width:$label_width;'>Next level in:</label> " . $exp_remaining . " exp<br />
 		<div style='height:6px;width:200px;border-style:solid;border-width:1px;'>" .
 			"<div style='background-color:#FFD700;height:6px;width:" . $exp_width . "px;' /></div>" . "</div>" .
-		"<br />
-		<label style='width:$label_width;'>Gender:</label> $player->gender<br />
+		"<br />";
+		if($player->gender == "None") {
+			echo "<label style='width:$label_width;'>Gender:</label> $player->gender
+			(Won't display)

```

Comment:
I think the "(Won't display)" part is better to put on the gender selection/change screens, we can just leave this as "None"

Reply:
done